modules/ROOT/pages/migration-transports.adoc

```diff
@@ -68,20 +77,29 @@ references Java classes with custom code to provide the new functionality.
 </imap:connector>
 ----
 
-In Mule 4, this kind of customization through custom Java code is not supported. There are three possible approaches
-that can be taken to migrate such scenario, depending on what the custom code is actually doing:
-
-* Replace it with some other feature of Mule or another component (i.e.: xref:dataweave.adoc[Data Weave] transformation, xref:connectors::java/java-module.adoc[Java Module]
-or xref:connectors::scripting/scripting-module.adoc[Scripting Module]) that provides a similar functionality.
-* Create a new connector based on the replacement of the modified transport with the custom code added. This will
-require working with the SDK that was used to implemented the new connector for Mule 4. Refer to xref:1.1@mule-sdk::index.adoc[Mule SDK]. Open Source connectors may be used as dependencies in order to have a starting point.
-* Similar to the previous approach, you may fork the connector and add the custom code there. Sending a Pull Request with that change is welcome, also.
-* You can request a feature to be added to a connector that does what your custom code did in `service-overrides`. Visit https://support.mulesoft.com/s/ideas[MuleSoft Support Ideas] to submit your requirement.
+The way to migrate such customizations to Mule 4 depends on what the custom
+code does, so you might:
+
+* Provide the functionality through another Mule feature or component, such as
+a xref:dataweave.adoc[DataWeave] transformation,
+a xref:connectors::java/java-module.adoc[Java Module],
+or a xref:connectors::scripting/scripting-module.adoc[Scripting Module]).
+* Create a new connector based on the replacement of the modified transport
+with the custom code added. This approach requires working with the SDK that
```

Comment:
Are we suggesting that they use the Mule SDK to create a custom connector that provides the customized transport functionality?

Reply:
yes